build.sbt

```diff
@@ -870,7 +870,8 @@ lazy val complianceTests = projectMatrix
         Dependencies.Pprint.core.value,
         Dependencies.Fs2Data.xml.value
       )
-    }
+    },
```

Comment:
**thought:** I like trailing commas but I don't think we've configured them, so this may have to be reverted

Reply:
done